device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/context_config.json

```diff
@@ -0,0 +1,66 @@
+{
+    "CONTEXTS": [
+        {
+            "guid" : 0,
+            "name" : "asic0",
+            "dbAsic" : "ASIC_DB",
+            "dbCounters" : "COUNTERS_DB",
+            "dbFlex": "FLEX_COUNTER_DB",
+            "dbState" : "STATE_DB",
+            "zmq_enable" : false,
+            "zmq_endpoint": "tcp://127.0.0.1:5565",
+            "zmq_ntf_endpoint": "tcp://127.0.0.1:5566",
+            "switches": [
+                {
+                    "index" : 0,
+                    "hwinfo" : ""
+                }
+            ]
+        },
+        {
+            "guid" : 1,
+            "name" : "phys",
+            "dbAsic" : "GB_ASIC_DB",
+            "dbCounters" : "GB_COUNTERS_DB",
+            "dbFlex": "GB_FLEX_COUNTER_DB",
+            "dbState" : "STATE_DB",
+            "zmq_enable" : false,
+            "zmq_endpoint": "tcp://127.0.0.1:5565",
+            "zmq_ntf_endpoint": "tcp://127.0.0.1:5566",
+            "switches": [
+                {
+                    "index" : 0,
+                    "hwinfo" : "mdio0_0_0/0"
+                },
+                {
+                    "index" : 1,
+                    "hwinfo" : "mdio1_0_0/0"
+                },
+                {
+                    "index" : 2,
+                    "hwinfo" : "mdio2_0_0/0"
+                },
+                {
+                    "index" : 3,
+                    "hwinfo" : "mdio3_0_0/0"
+                },
+                {
+                    "index" : 4,
+                    "hwinfo" : "mdio4_0_0/0"
+                },
+                {
+                    "index" : 5,
+                    "hwinfo" : "mdio5_0_0/0"
+                },
+                {
+                    "index" : 6,
+                    "hwinfo" : "mdio6_0_0/0"
+                },
+                {
+                    "index" : 7,
+                    "hwinfo" : "mdio7_0_0/0"
+                }
+            ]
```

Comment:
Are all 8 PHYs (0 - 7) belonging to same NPU/ASIC? 

Is/Are there PHY(s) which needs to be on another NPU? 
In that case, would it be following SONiC multi-NPU/multi-ASIC architecture ? (i.e. have these gearbox_config, phy<x>_config_1 , context_config json files under /usr/share/sonic/device/swss@<NPU> syncd@<NPU>  and gbsyncd@<NPU> )?

Reply:
For this specific platform, the setting is 8 external phy chips connected 1 ASIC.
So far for the platforms we planned to support, we haven't considered about the multi-ASIC case.